tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Redis/StackExchange/RedisExecuteAsyncIntegration_2_6_48.cs

```diff
@@ -0,0 +1,62 @@
+// <copyright file="RedisExecuteAsyncIntegration_2_6_48.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Redis.StackExchange
+{
+    /// <summary>
+    /// StackExchange.Redis.[RedisBatch/RedisTransaction].ExecuteAsync[T] calltarget instrumentation for 2.6.48+
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyNames = new[] { "StackExchange.Redis", "StackExchange.Redis.StrongName" },
+        MethodName = "ExecuteAsync",
+        ReturnTypeName = "System.Threading.Tasks.Task`1<T>",
+        ParameterTypeNames = new[] { "StackExchange.Redis.Message", "StackExchange.Redis.ResultProcessor`1[!!0]", "!!0", "StackExchange.Redis.ServerEndPoint" },
+        MinimumVersion = "2.0.0",  // 2.6.48, but dll uses 2.0.0
+        MaximumVersion = "2.*.*",
+        TypeNames = new[] { "StackExchange.Redis.RedisBatch", "StackExchange.Redis.RedisTransaction" },
+        IntegrationName = StackExchangeRedisHelper.IntegrationName)]
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    // ReSharper disable once InconsistentNaming
+    public class RedisExecuteAsyncIntegration_2_6_48
+    {
+        internal static CallTargetState OnMethodBegin<TTarget, TMessage, TDefaultValue, TProcessor, TServerEndPoint>(TTarget instance, TMessage message, TDefaultValue defaultValue, TProcessor resultProcessor, TServerEndPoint serverEndPoint)
+            where TTarget : IRedisBase
+            where TMessage : IMessageData
+        {
+            string rawCommand = message.CommandAndKey ?? "COMMAND";
+            StackExchangeRedisHelper.HostAndPort hostAndPort = StackExchangeRedisHelper.GetHostAndPort(instance.Multiplexer.Configuration);
+
+            Scope scope = RedisHelper.CreateScope(Tracer.Instance, StackExchangeRedisHelper.IntegrationId, hostAndPort.Host, hostAndPort.Port, rawCommand);
+            if (scope is not null)
```

Comment:
I can already imagine my rider not happy about not using `var` and not using a single return statement :trollface:
Not asking to change anything, IIRC we have the same code for all classes in this instrumentation

Reply:
> I can already imagine my rider not happy about not using `var`

Yeah, mine too 😂 I copy pasted this from the existing `RedisExecuteAsync` integration and thought it better that they stay the same rather than do any cleanup around that :wink: